### DISCUSSION TITLE
Add Confluence RAG Tool

### DISCUSSION THREAD

**Discussion by samuelbirocchi** [langgenius/dify#3735] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Yes we need to retrieve the data from the Jira Service Desk help center, which is in a space within Confluence

### 2. Describe the feature you'd like to see

Retrieve data from a Confluence space, and configure how the data will be embedded in the vector db

### 3. How will this feature improve your workflow or experience?

I can do everything I need inside dify

### 5. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by gabilanbrc** [comment]
Hi! Can I help somehow on implementing this retrieval from Confluence?

**Comment by lostmaniac** [comment]
+1

**Comment by nhlzh8708** [comment]
+1

**Reply by o0vvv0o** [reply]
+1

**Reply by kylianhe** [reply]
+1

**Reply by Ian879** [reply]
+1

**Comment by paranoiagu** [comment]
+1

**Comment by wenyongda** [comment]
+1

**Comment by fdb02983rhy** [comment]
A external rag pipeline will be much easier, which is what I did.

**Comment by kyuchong** [comment]
flowise 처럼 confluence loader 지원이 필요합니다.
